layouts/_default/baseof.html

```diff
@@ -1,5 +1,5 @@
-<!DOCTYPE html>
-<html>
+<!DOCTYPE html >
```

Comment:
```suggestion
<!DOCTYPE html>
```
I am assuming this was changed by accident?

Reply:
yep, i fixed it in the next commit again, sry